gluoncv/loss.py

```diff
@@ -76,6 +76,10 @@ def hybrid_forward(self, F, pred, label, sample_weight=None):
             pred = F.sigmoid(pred)
         if self._sparse_label:
             one_hot = F.one_hot(label, self._num_class)
+            ndims = len(pred.shape)
```

Comment:
hybridblock(F is symbol) don't have `shape` property, so this is not a valid hybridBlock, but normal Block

Reply:
Yes, my fault

Do you know how this issue can be solved?
Maybe it's a good idea to update the documentation and highlight that the axis parameter is not supported now?

Reply:
Yes, I agree, it's a bit tricky so can you update the doc accordingly? We will try to find a better workaround to fix it.